### PR TITLE
Add Azure and Log Analytics setup instructions

### PR DESCRIPTION
**Pull request by kylesmartin**
I added instructions to docs/aks/README.md for setting up a Log Analytics workspace and getting the proper credentials.

### CODE REVIEW

File: docs/aks/README.md [hunk 1]
```diff
@@ -15,22 +15,17 @@ Make sure only the following permissions are assigned to the key pair:
...
-```
-WORKSPACE_ID=ASDF
-```
+In your Azure portal, create a new resource group. (Home > Resource groups > Create)
 
-```
-PRIMARY_KEY=ASDF
-```
-### Step 3: Set-up Azure Log Analytics
+Create a new Log Analytics workspace and assign it to your resource group. (Home > Log Analytics workspaces > Create)
```
Comment (redhatrises) [inline docs/aks/README.md:24]:
```suggestion
1. Create a new Log Analytics workspace by selecting `Log Analytics workspaces`. You might have to search for it as a resource if it is not in your home menu.
1. Select `New`
1. Select the resource group that was previously created and enter a name under `Instance details`
1. Select `Review + Create`
1. Select `Create`

ssign it to your resource group. (Home > Log Analytics workspaces > Create)
```

**Review by redhatrises — CHANGES_REQUESTED** [review]

**Review by isimluk — APPROVED** [review]
Thanks!

File: docs/aks/README.md (hunk 1 above)
Comment (isimluk) [reply docs/aks/README.md:24]: `ssign` above may be a typo

**Comment by isimluk** [comment]
Guys, I am eager to merge this. Both of your versions are fine with me. :four_leaf_clover:

File: docs/aks/README.md (hunk 1 above)
Comment (kylesmartin) [reply docs/aks/README.md]: Fixed the typo

**Comment by kylesmartin** [comment]
I think everything is good to merge now.

File: docs/aks/README.md [hunk 2]
```diff
@@ -15,22 +15,27 @@ Make sure only the following permissions are assigned to the key pair:
...
-PRIMARY_KEY=ASDF
-```
-### Step 3: Set-up Azure Log Analytics
+1. Create a new Log Analytics workspace by selecting `Log Analytics workspaces`. You might have to search for it as a resource if it is not in your home menu.
+1. Select `New`
+1. Select the resource group that was previously created and enter a name under `Instance details`
+1. Select `Review + Create`
+1. Select `Create`
+
+Assign it to your resource group. (Home > Log Analytics workspaces > Create)
```
Comment (redhatrises) [inline docs/aks/README.md:34]: I think these lines need to be dropped. They appear to be duplicates of the instructions in line 28-32, but once they are, I think we should merge!

**Pull request merged**